package.json

```diff
@@ -18,13 +18,14 @@
     "pull-stream": "^3.6.14",
     "private-group-spec": "^2.1.1",
     "ssb-bfe": "^3.7.0",
-    "ssb-keyring": "^3.0.0",
+    "ssb-keyring": "https://gitlab.com/powersource/ssb-keyring/-/archive/live-list/ssb-keyring-live-list.tar",
```

Comment:
need to merge and publish https://gitlab.com/ahau/lib/ssb-keyring/-/merge_requests/19